tracer/test/Datadog.Trace.Security.Unit.Tests/WafCompatibilityTests.cs

```diff
@@ -18,7 +18,6 @@ public void ShouldNotInitialize()
             SkipOn.PlatformAndArchitecture(SkipOn.PlatformValue.MacOs, SkipOn.ArchitectureValue.ARM64);
             var libraryInitializationResult = WafLibraryInvoker.Initialize("1.4.0");
             libraryInitializationResult.Success.Should().BeFalse();
-            libraryInitializationResult.ExportErrorHappened.Should().BeTrue();
```

Comment:
why is this line removed?

Reply:
Because the function missing in that version does not exist in the new one (`ddwaf_ruleset_info_free`), so there was no ExportError...

Reply:
you can replace it with 
          ` var libraryInitializationResult = WafLibraryInvoker.Initialize("1.3.0");`
this should be missing exports (tested locally)

Reply:
In the end, I opened [a small PR for waf incompatibility tests](https://github.com/DataDog/dd-trace-dotnet/pull/4368), feel free to integrate / modify

Reply:
PR merged 👍